Spigot-Server-Patches/0252-Prevent-logins-from-being-processed-when-the-player-.patch

```diff
@@ -0,0 +1,27 @@
+From 73bf7eb9bf32feb050447647cce710b2f1fa7064 Mon Sep 17 00:00:00 2001
+From: killme <killme-git@ibts.me>
+Date: Sun, 12 Nov 2017 19:40:01 +0100
+Subject: [PATCH] Prevent logins from being processed when the player has
+ disconnected
+
+
+diff --git a/src/main/java/net/minecraft/server/LoginListener.java b/src/main/java/net/minecraft/server/LoginListener.java
+index 75df928..ccbae2f 100644
+--- a/src/main/java/net/minecraft/server/LoginListener.java
++++ b/src/main/java/net/minecraft/server/LoginListener.java
+@@ -60,7 +60,11 @@ public class LoginListener implements PacketLoginInListener, ITickable {
+         }
+         // Paper end
+         if (this.g == LoginListener.EnumProtocolState.READY_TO_ACCEPT) {
+-            this.b();
++            // Paper start - prevent logins to be processed even though disconnect was called
++            if (networkManager.isOpen()) {
```

Comment:
`isConnected()`

Reply:
I really should have actually checked this

Reply:
Fixed